### PR TITLE
Project entity

### PR DESCRIPTION
**Pull request by hys25**

### CODE REVIEW

**Comment by hys25** [comment]
Hi @ArthurGorbenko, pushed changes on hamepage, project amd skils pages. Please check when you have time

File: components/elements/input.js [hunk 1]
```diff
@@ -77,16 +76,11 @@ export function FormInput({
...
 }
 
-export function PasswordInput({
-  setValue,
-  className,
-  password = false,
-  ...props
-}) {
-  const [type, setType] = useState("password");
+export function PasswordInput({ setValue, className, ...props }) {
```
Comment (ArturHorbenko) [inline components/elements/input.js]: I would suggest to revise elements and remove unused code from `input.js` for example.

File: pages/about/index.js [hunk 2]
```diff
@@ -1,7 +1,59 @@
-import { Fragment } from "react";
-import Default from "../../components/layout/default";
+import Link from "next/link"
+import { get } from "../../lib/api"
+import Default from "../../components/layout/default"
 
-function About() {
-  return <Default />;
+export async function getServerSideProps() {
+  const { data, message, isError } = await get("http://localhost:3000/skill")
```
Comment (ArturHorbenko) [inline pages/about/index.js]: Let's move api url to environment variable

File: pages/index.js [hunk 3]
```diff
@@ -1,87 +1,114 @@
...
+  }
+}
+
+function Homepage({ projects, message, isError }) {
+  const mainProjects = projects.filter(
+    (project) => project.main_project === true
+  )
+  const otherProjects = projects.filter(
+    (project) => project.main_project === false
+  )
```
Comment (ArturHorbenko) [inline pages/index.js]: Let's do some optimization here 
- wrap computations in `useMemo`
- loop over the array only once and either push in `mainProjects` array, either in `otherProjects`, then return both arrays from `useMemo`

File: pages/project/index.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
+import { Fragment } from "react"
+import ReactFullpage from "@fullpage/react-fullpage"
+import { get } from "../../lib/api"
+import Sidebar from "../../components/modules/sidebar"
+import Footer from "../../components/modules/footer"
+import { normalizeAnchor } from "../../lib/utils"
+
+export async function getServerSideProps() {
+  const { data, message, isError } = await get("http://localhost:3000/project")
```
Comment (ArturHorbenko) [inline pages/project/index.js]: Same here, let's move BE host to env.

**Review by ArturHorbenko — APPROVED** [review]
Hi, great work !
I left couple comments about env variables and optimization.

**Comment by hys25** [comment]
@ArthurGorbenko Thank you a lot for a review, pushed fix

**Comment by ArturHorbenko** [comment]
LGTM!

**Pull request merged**